content/docs/authorization/cli.md

```diff
@@ -908,15 +909,16 @@ karavictl tenant create [flags]
 ##### Options
 
 ```
+  -a, --approvesdc    To allow/deny SDC approval requests (default true)
```

Comment:
We should still indicate this is applicable only for PowerFlex. https://dell.github.io/csm-docs/docs/authorization/cli/#options-25

Reply:
Fixed.